### ISSUE TITLE
VSCode suggestions are not helpful after update

### ISSUE THREAD

**Issue by naeem-hasan** [flutter/flutter#32572]
Hello! After upgrading to the latest version(1.54 I believe), the suggestions/auto-completes in VSCode are not working as they did before.
![image](https://user-images.githubusercontent.com/24958892/57577908-52d9af80-74a3-11e9-8337-351426dca44c.png)
In the given screenshot, it should show a list of widgets, I believe. But even after I type the whole "Container", what I want doesn't show. How can I resolve this?

**Comment by naeem-hasan** [comment]
I restarted the IDE, it's working as expected now. I don't know why that happened.

**Comment by gkrawiec** [comment]
I was about to post the following before I found this message. The vscode restart fixed it. Is this a vscode bug or a flutter extension bug?
----------
This might be related to the last vscode update and suggestions now also coming from other modules, but now suggestions almost never suggest Widgets.
For example if I type "Containe" the suggestion list will still not suggest "Container()"
![Screenshot 2019-05-14 19 52 17](https://user-images.githubusercontent.com/645451/57737358-d05b1680-7681-11e9-907a-a6108e376419.png)
Here is MaterialApp()
![Screenshot 2019-05-14 19 51 53](https://user-images.githubusercontent.com/645451/57737366-d4873400-7681-11e9-9c94-ce9121df0aec.png)
Row
![Screenshot 2019-05-14 19 56 42](https://user-images.githubusercontent.com/645451/57737544-6f800e00-7682-11e9-8fbe-3c7c57ccc6f7.png)
but the same problem happens with all widgets I've tried. If I complete it myself, then the properties are suggested correctly
![Screenshot 2019-05-14 19 54 09](https://user-images.githubusercontent.com/645451/57737422-0bf5e080-7682-11e9-9dfd-40f4b3883bc0.png)
